qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -93,6 +93,9 @@
                             cpu_model_flags += ",+pcid,+invpcid"
         - others:
             variants:
+                - test_default_model:
+                    auto_cpu_model = no
```

Comment:
Would you pls add a comment here, such as "will use the default_cpu_model defined in avocado-vt xxx.cfg, need update it if default model changes in the future".

Reply:
> Would you pls add a comment here, such as "will use the default_cpu_model defined in avocado-vt xxx.cfg, need update it if default model changes in the future".

Thanks and I updated it using a simple so we can search it later.